src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
 package library.returnBook;
+
 import java.util.Scanner;
 
 
 public class ReturnBookUI {
 
-    private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    private enum UiState {INITIALISED, READY, INSPECTING, COMPLETED};
+
+    private ReturnItemControl control;
+
+    private Scanner input;
 
-    private ReturnItemControl CoNtRoL;
-    private Scanner iNpUt;
-    private uI_sTaTe StATe;
+    private UiState state;
 
-    
-    public ReturnBookUI(ReturnItemControl cOnTrOL) {
-        this.CoNtRoL = cOnTrOL;
-        iNpUt = new Scanner(System.in);
-        StATe = uI_sTaTe.INITIALISED;
-        cOnTrOL.setUI(this);
+
+    public ReturnBookUI(ReturnItemControl control) {
```

Comment:
#PR - D5: Please rename constructor to ReturnItemUI (probably will be fixed when refactoring class/file name)

Reply:
https://github.com/elliotmitchum/ITC205/pull/42/commits/8b325666fce4f04d4977b7d4dd50069c5cf81a7c